15-lhcb-dev.md

```diff
@@ -76,16 +76,41 @@ It works similar to `lb-run`, without the need to specify a package and version:
 
 > ## What if getpack asks for my password 1000 times? {.callout}
 > `getpack` might ask you for your password several times.
-> To avoid this, you can create a kerberos token with
-> ```
-> kinit
+> To avoid this, you need to properly configure your `ssh` following the
```

Comment:
Can we change this to something like "you need to configure ssh to use public key login". Not sure what exactly the right terminology is, but we should use that.